src/PhpSpec/CodeAnalysis/TokenizedTypeHintRewriter.php

```diff
@@ -192,4 +197,25 @@ private function tokenHasType($token, $type)
     {
         return is_array($token) && $type == $token[0];
     }
+
+    /**
+     * @param string $className
+     *
+     * @return bool
+     */
+    private function shouldExtractTokensOfClass($className)
+    {
+        return substr($className, -4) == 'Spec';
```

Comment:
This duplicates logic inside the Locator (i.e. it is an assumption that an extension could change). I'm not really happy having it here.